<?xml version="1.0"?>
<launch>
    <node pkg="nakoon_element" type="nakoon_element_node" name="nakoon_element" respawn="false" output="log">
        <param name="port"                 type="string"  value="/dev/ttyS0"/>
        <param name="baud_rate"            type="int"     value="38400"/>
        <param name="publish_tf"           type="bool"    value="false"/>
        <param name="max_trans_velocity"   type="double"  value="0.2"/>
        <param name="max_rot_velocity"     type="double"  value="0.4"/>
        <param name="wheel_base"           type="double"  value="0.47"/>
        <param name="velocity_raw_factor"  type="double"  value="300.0"/>
        <param name="raw_odometry_factor"  type="double"  value="0.0000016129"/>
        <param name="frame_id"             type="string"  value="/odom"/>
        <param name="child_frame_id"       type="string"  value="/robot_footprint"/>
    </node>

    <!-- static transform from odom to robot_footprint (if publish_tf == false)-->
    <node pkg="tf" type="static_transform_publisher" name="odom_to_robot_footprint" args="0 0 0 0 0 0 odom robot_footprint 50" />
</launch> 
